framework/src/main/java/org/tron/core/services/http/solidity/GetTransactionByIdSolidityServlet.java

```diff
@@ -41,7 +46,12 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
       JsonFormat.merge(params.getParams(), build, params.isVisible());
       fillResponse(build.build().getValue(), params.isVisible(), response);
     } catch (Exception e) {
-      Util.processError(e, response);
+      logger.debug("Exception: {}", e.getMessage());
+      try {
+        response.getWriter().println(e.getMessage());
+      } catch (IOException ioe) {
+        logger.debug("IOException: {}", ioe.getMessage());
+      }
```

Comment:
Maybe it's not a good idea to modify the error message and remove Exception for every API. A better solution would be to update the implementation of Util.processError uniformly.

Reply:
@317787106 This is a legacy API; do not modify anything, even the error messages.

Reply:
Get it